web_src/js/features/comp/QuickSubmit.js

```diff
@@ -8,7 +8,7 @@ export function handleGlobalEnterQuickSubmit(target) {
 
     // here use the event to trigger the submit event (instead of calling `submit()` method directly)
     // otherwise the `areYouSure` handler won't be executed, then there will be an annoying "confirm to leave" dialog
-    form.dispatchEvent(new SubmitEvent('submit', {bubbles: true, cancelable: true}));
+    form.dispatchEvent(new SubmitEvent('submit'));
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/Event

![image](https://github.com/go-gitea/gitea/assets/2114189/d89ff6e4-df0e-4217-b97d-81c62a366560)

Reply:
Yes, but does that also apply to `SubmitEvent`? Neither MDN nor the HTML spec seem to give an answer, so we may need to test it.

Reply:
I was lied to
![image](https://github.com/go-gitea/gitea/assets/20454870/ec00b803-f67e-4eca-90b0-e43fd6c22722)